### PR TITLE
Add a little code to prepare exception handling support in fuzzer

### PR DESCRIPTION
**Pull request by kripken**
`fixLabels()` in the fuzzer looks for invalid labels and fixes them up,
after doing some random changes to existing wasm (which checks
for types while doing so, but it may invalidate labels if we remove the
target of a branch, for example). This adds trivial support for
`BrOnExn` and `Try` there.

I wasn't working on exceptions, just a fuzz testcase happened to hit
this, and this was enough to get past it.

### CODE REVIEW

**Review by aheejin — APPROVED** [review]
Thanks! This was also included in #2880, but I was holding off on landing it because I wasn't able to debug a fuzzer failure on `CheckDeterminism` found after like a million iterations ;(

**Comment by kripken** [comment]
@aheejin Ok, let's land this for now, but if the determinism issue in the other PR is that rare, maybe we can land it too? As this is an experimental feature anyhow. We could also disable the determinism check in the fuzzer when exceptions are enabled.

**Comment by aheejin** [comment]
We have to land a major change soon, and having the fuzzer on makes it difficult. If you agree with the timeline that "enable EH fuzzer" -> "disable the fuzzer for EH before we land the major change" -> "Add EH changes piece by piece" -> "Once everything is done reenable the EH fuzzer", that might work.

**Comment by kripken** [comment]
@aheejin oh, it sounds fine to me either way. I'm doing some experiments locally with turning more things in the fuzzer, but there is no rush to do so in general. can wait for whenever things stabilize.

**Pull request merged**
